templates/components/navbar.jinja2

```diff
@@ -1,35 +1,43 @@
 <header>
-    <nav class="navbar navbar-expand-md navbar-light bg-light">
-        <div class="container">
+    <nav class="navbar navbar-expand-lg bg-light" data-bs-theme="light">
+        <div class="container-fluid">
             <a class="navbar-brand" href="/"><img src="/static/images/logo.png" class="logo"></a>
-            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarColor01" aria-controls="navbarColor01" aria-expanded="false" aria-label="Toggle navigation">
                 <span class="navbar-toggler-icon"></span>
             </button>
-            <div class="collapse navbar-collapse">
-                <ul class="navbar-nav mr-auto">
+            <div class="collapse navbar-collapse" id="navbarColor01">
+                <ul class="navbar-nav me-auto">
                     <li class="nav-item">
-                        <a class="nav-link" href="/add">Add targets</a>
+                        <a class="nav-link {{ 'active' if request.path == '/add' }}" href="/add">Add targets</a>
                     </li>
                     <li class="nav-item">
-                        <a class="nav-link" href="/">View targets</a>
+                        <a class="nav-link {{ 'active' if request.path == '/' }}" href="/">View targets</a>
                     </li>
                     <li class="nav-item">
-                        <a class="nav-link" href="/results?task_filter=interesting">View raw task results</a>
+                        <a class="nav-link {{ 'active' if request.path == '/results' }}" href="/results?task_filter=interesting">View raw task results</a>
```

Comment:
would it work if the path is e.g. /results?task_filter=interesting ?

Reply:
yes, already tested